Libraries/src/Amazon.Lambda.Logging.AspNetCore/Amazon.Lambda.Logging.AspNetCore.csproj

```diff
@@ -17,9 +17,9 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="2.1.0" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="2.1.0" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="2.1.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="[3.1.0,3.2.0)" />
```

Comment:
This library still needs to work with ASP.NET Core 2.1 Lambda functions so i don't want to bump the version to 3.1 as the minimum. Is there a reason we need to do that? Also what is the goal of the using a range blocking at 3.2? That would prevent usage for people that want to use this with .NET 5 preview using custom runtimes.

I think we can just take this part out of the PR and then release the NullScope fix.

Reply:
@normj that's the point of this PR:

- when on `netcore2.x` -> reference `Amazon.Lambda.Logging.AspNetCore` version `3.0.0` 
- when on `netcore3.x` -> reference `Amazon.Lambda.Logging.AspNetCore` version `3.1.0` 

if there are people on 5, we should of course provide another version for it.

The best would be to split code into branches based on runtime version. We should avoid mixing major versions - if I run `netcore3.1` I don't want to see dependency targeting `netcore2.1`, because it can have side effect elsewhere.

Reply:
Why would we want .NET Core 2.1 users use the versions that came for .NET Core 3.0? In general by targeting a lower version it will work for versions going forward. For example if you look at the Serilog log provider it still references version 2.0.0 of Microsoft.Extensions.Logging.

We would only need to create a separate build for the newer runtimes if we were using features that were only available in the newer runtimes.

In this case we made a poor decision by using a type that was in an "internal" namespace. And that is the part of the PR I want to take which fixes that.

Reply:
Oh, you are right - all of these are `netstandard`! I will revert that change and keep only `NullScope` thingy.

In other hand, what do you think about version range? What happens if my app depends on another package which references different version of one of those packages? (Basically what happened to me with logging)